### ISSUE TITLE
500 Internal Server Error

### ISSUE THREAD

**Issue by mdbetancourt** [styfle/packagephobia#877]
package with dependencies as "typescript@beta" make the website throw error (example package @kraftr/errors)

**Comment by styfle** [comment]
The error logs says `@kraftr/errors` doesn't exist 🤔 

```
Error: Package "@kraftr/errors" was not found
ERROR	Unexpected Error Occurred... Error: Package "@kraftr/errors" does not exist in npm
```

**Comment by styfle** [comment]
Oh its because its missing the `description` prop.

This will be fixed in #878
